### ISSUE TITLE
Make icons more accessible

### ISSUE THREAD

**Issue by Rabadash8820** [Rabadash8820/Dansite#20]
As recommended in the [Bootstrap `icon-link` docs](https://getbootstrap.com/docs/5.3/helpers/icon-link/):

> When icons are purely decorative, they should be hidden from assistive technologies using `aria-hidden="true"`... For icons that convey meaning, provide an appropriate text alternative by adding `role="img"` and an appropriate `aria-label="..."` to the SVGs.

So make sure to follow these guidelines:

- [x] Project category nav tab icons: decorative (done in b837d85d20c9229f4089bb235e96c9250eab0eee)
- [ ] Main contact links on home page: semantic
- [ ] Footer contact links: semantic
- [x] Project external links: decorative (done in b837d85d20c9229f4089bb235e96c9250eab0eee)
- [ ] Main nav icons: semantic
- [x] Color mode icons: decorative (done in b837d85d20c9229f4089bb235e96c9250eab0eee)
- [ ] Scroll indicator: semantic
- [ ] Carousel indicators: semantic
